Ransomwares/HIve/Inhibit System Recovery/delete_backup_using_vssadmin.md

```diff
@@ -1,9 +0,0 @@
-# Hive
```

Comment:
any reason we removed this query..

Reply:
I renamed the file to delete_volume_shadow_copies_and_disable_recovery.md and added a new query for "Windows recovery disabling events"